src/DataDryadAPI.jl

```diff
@@ -0,0 +1,67 @@
+immutable DataDryadAPI <: DataRepo
+end
+
+base_url(::DataDryadAPI) = "https://datadryad.org/mn/object/http://dx.doi.org/"
+
+function description(repo::DataDryadAPI, mainpage)
+    desc = text_only(first(matchall(sel"dcterms\:description", mainpage.root)))
+    authors = matchall(sel"dcterms\:creator", mainpage.root)
+    author = text_only(first(authors))
+    license = "http://creativecommons.org/publicdomain/zero/1.0/"
+    rawdate = chop(text_only(first(matchall(sel"dcterms\:dateSubmitted", mainpage.root))))
+    year = Dates.year(Dates.DateTime(rawdate))
+    month = Dates.monthname(Dates.DateTime(rawdate))
+    day = Dates.day(Dates.DateTime(rawdate))
+    date = "$(month) $(day), $(year)"
```

Comment:
You can format this a bit more cleanly using `date.format`
https://docs.julialang.org/en/stable/manual/dates/

Something like
```
date = Dates.DateTime(rawdate)
Dates.format(dt, "mm dd, yyyy")
year = Dates.year(date)
```

I think